src/qz/ui/ThemeUtilities.java

```diff
@@ -0,0 +1,46 @@
+package qz.ui;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ThemeUtilities {
```

Comment:
I propose the following changes to the `refreshAll` methods in this class

```java
public static void refreshAll(Container container, Component ... stragglers) {
    // Handle orphaned UI objects (e.g. Component added to a message dialog)
    for(Component straggler : stragglers) {
        /// as is
    }

    refreshAll(ArrayUtils.addAll(c.getComponents(), stragglers));
}

public static void refreshAll(Component ... components) {
    for(Component c : components) {
        /// as is
    }
}
```

Unless there's a reason to refresh the container explicitly before the stragglers, in which case then I think we should get rid of the first overload (with passing the null) and the associated null checks and just let the for-loops iterate over the resulting 0 element arrays.

Reply:
Done via d39792d.